Embedded System L1/AVR ATMEGA32/DIO/PROJECT/DIO_interface.h

```diff
@@ -0,0 +1,91 @@
+/******************************************************************************************************
+ * File: DIO_program.h 
+ * Author: Khalid Mosaad Zakaria
+ * Data: 10 july 2021
+ * Description: This File include Declarations of functions and Strucrures and some Macros.
+*******************************************************************************************************/
+
+#ifndef  DIO_interface_h_
+#define  DIO_interface_h_
+
+#define		PORTA	(u8)0
+#define		PORTB	(u8)1
+#define		PORTC	(u8)2
+#define		PORTD	(u8)3
+
+
+
+#define 	PIN0	(u8)0
+#define 	PIN1	(u8)1
+#define 	PIN2	(u8)2
+#define 	PIN3	(u8)3
+#define 	PIN4	(u8)4
+#define 	PIN5	(u8)5
+#define 	PIN6	(u8)6
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of headers

![2](https://user-images.githubusercontent.com/44152298/127502084-967ab9b6-4fba-4be7-ba83-0af348f54c31.PNG)

[Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)